common/alert/reports/a11y-1.json

```diff
@@ -0,0 +1,847 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+  "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/common/alert/en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "",
+    "earl:pointer": {
+      "@type": "",
+      "@value": ""
+    }
+  },
+
+  "earl:assertedBy": {
+    "foaf:name": "Service Canada - Principal Publisher",
+    "foaf:homepage": "https://github.com/ServiceCanada",
+    "@type": ["earl:Assertor", "foaf:Organization"],
+    "earl:mainAssertor": {
+      "foaf:name": "Service Canada - Principal Publisher",
```

Comment:
Replace the value of this [foaf:name](http://xmlns.com/foaf/0.1/#term_name) for the [earl:mainAssertor](https://www.w3.org/TR/EARL/#mainAssertor) by your name